pica2mods-xslt/src/main/resources/xsl/default/pica2mods-default-identifier.xsl

```diff
@@ -210,7 +210,7 @@
 
     <!-- URLs -->
     <xsl:for-each
-      select="./p:datafield[@tag='017C' and (./p:subfield[@code='x']='D' or ./p:subfield[@code='x']='H')]/p:subfield[@code='u']">
+      select="./p:datafield[@tag='017C' and (./p:subfield[@code='x']='D' or ./p:subfield[@code='x']='T' or ./p:subfield[@code='x']='H')]/p:subfield[@code='u']">
```

Comment:
Mit $x=T werden gemäß GBV-Format-Richtlinie Links zu DBIS (Datenbank-Informationssystem) erfasst.
Ich bin mir unsicher, ob wir die als "normale URL" behandeln sollten.

Ich könnte da eher mit einem mods:identifier[@type='dbis_url'] leben.
Der Link ist außerdem nicht stabil und wird bereits weitergeleitet:
http://dbis.uni-regensburg.de/frontdoor.php?titel_id=12377
-> https://dbis.ur.de/frontdoor.php?titel_id=12377

oder `<mods:identifier @type='dbis'>12377</mods:identifier>`

Unter `type='url'` sollte nur Links auf die Ressource selbst (beim Anbieter, Verlag, ...) erfasst werden.

Hohl mal bei Kathleen eine Zweitmeinung - ich mach erstmal weiter Urlaub ;-)

Reply:
Also ich hab mir das jetzt mal in Ruhe angeschaut und plädiere für mods:identifier. Wenn es wirklich nur "die DBIS-Datenbank" gibt, dann passt das so besser. Die Anzeige sollte dann in der Anwendung gelöst werden und dort auch die entsprechende URL anhand des Identifiers erzeugt werden.

Reply:
Offenbar soll nur die URL zur Landing Page (zur weiteren Recherche übernommen werden)

`  <datafield tag="017C">
    <subfield code="u">https://urbrel.hypotheses.org/</subfield>
    <subfield code="x">H</subfield>
    <subfield code="4">LF</subfield>
  </datafield>
`

Da könnten wir die Klärung des Umgangs mit der DBIS-URL erstmal verschieben?